modules/support-reviewing-an-access-request-from-the-hybrid-console.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * support/getting-support.adoc
+// * osd_architecture/osd-support.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="support-reviewing-an-access-request-from-the-hybrid-cloud-console_{context}"]
+= Reviewing an access request from the {hybrid-console-second}
+
+.Prerequisites
+
+ifndef::openshift-rosa,openshift-dedicated[]
+* You have access to the cluster as a user with the `cluster-admin` role.
+endif::openshift-rosa,openshift-dedicated[]
+
+
+.Procedure
+
+. Navigate to link:https://console.redhat.com/openshift[Red{nbsp}Hat OpenShift Cluster Manager] and select *Clusters*.
+
+. Click the cluster *Name* to review the *Access Request*.
+
+. Select the *Access Requests* tab to list all *states*.
+
+. Select *Open* under *Actions* for the *Pending* state.
+
+. Click *Approve* or *Deny* under *Decision* in the *Access Request Details* modal.
++
+[NOTE]
+====
+Denying an access request will require you to complete the *Justification* field and will require further support case engagement, to help troubleshoot the issue, given that a Red{nbsp}Hat SRE would not be able to directly act on the resources related to the support request.
```

Comment:
The sentence `will require further support case engagement` seems unnecessary, as it's not really a requirement. I'd phrase this whole note with something like:

```
Denying an access request will require you to complete the Justification field. In this case, Red Hat SRE will not be able to directly act on the resources related to the incident. Customers can still use the Red Hat Support to help investigate and fix the issue.
```